Zattoo/Fastfile

```diff
@@ -0,0 +1,183 @@
+fastlane_version "1.91.0"
+
+require './helpers/xcode_project_helpers.rb'
+require './helpers/zattoo.rb'
+
+#### Setup ####
+
+desc "Fetches development certificates and provisioning profiles"
+lane :fetch_dev_certificates do
+  match(type: 'development', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::AppIdentifier::Production)
+  match(type: 'development', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::TopShelfAppIdentifier::Production)
+end
+
+desc "Fetches distribution certificates and provisioning profiles"
+lane :fetch_distribution_certificates do
+  match(type: 'appstore', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::AppIdentifier::Preview)
+  match(type: 'appstore', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::AppIdentifier::AppleStore)
+  match(type: 'appstore', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::AppIdentifier::Production)
+  match(type: 'appstore', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::TopShelfAppIdentifier::Preview)
+  match(type: 'appstore', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::TopShelfAppIdentifier::AppleStore)
+  match(type: 'appstore', readonly: true, team_id: Zattoo::TeamId, app_identifier: Zattoo::TopShelfAppIdentifier::Production)
+end
+
+desc "Update device list"
+lane :update_devices do
+  register_devices(devices_file: 'fastlane/devices.txt', team_id: Zattoo::TeamId)
+end
+
+desc "Install Xcode plugins"
+lane :install_xcode_plugins do
+  install_xcode_plugin(github: 'https://github.com/neonichu/FixCode')
```

Comment:
This is probably not part of the setup any more as this doesn't work any more, right?

Reply:
Sad world we live in :(

Reply:
Indeed, removed !!! 🗑 

Cheer up @KrauseFx at least it is Friday !!!

![bd6](https://user-images.githubusercontent.com/2320840/33482639-5a789d26-d69a-11e7-8849-919637fc7d36.gif)